dist/profile/manifests/ldap.pp

```diff
@@ -19,7 +19,10 @@
       'libaugeas-ruby',          # for augeas based puppet providers
   ])
 
-  include openldap::server
+  class { 'openldap::server':
+    ldap_ifs  => ['127.0.0.1'],
+    ldaps_ifs => ['/'],
```

Comment:
`ldapi` is [part of the module defaults](https://github.com/camptocamp/puppet-openldap/blob/master/manifests/server.pp#L24)

I'll add it as an explicit declaration in a followup pull request.